core/src/main/java/org/codice/imaging/nitf/core/image/impl/RasterProductFormatUtilities.java

```diff
@@ -59,25 +66,42 @@ public RasterProductFormatUtilities() throws NitfFormatException {
     }
 
     /**
-        Unmarshal input stream to RPFs list.
-
-        @param inputStream the input stream to parse.
-        @throws JAXBException if the unmarshalling of the XML fails.
-    */
+     * Unmarshal input stream to RPFs list.
+     *
+     * @param inputStream the input stream to parse.
+     * @throws JAXBException if the unmarshalling of the XML fails.
+     */
     private void unmarshal(final InputStream inputStream) throws JAXBException {
+        DocumentBuilderFactory dbf = DocumentBuilderFactory.newInstance();
+
+        Document document = null;
+        try {
+            dbf.setFeature("http://xml.org/sax/features/external-general-entities", false);
+            dbf.setFeature("http://xml.org/sax/features/external-parameter-entities", false);
+            dbf.setFeature("http://apache.org/xml/features/nonvalidating/load-external-dtd", false);
```

Comment:
Will an attempt be made by DocumentBuilderFactory to try to access these URLs? If it does, then this could be an issue in environments that don't have access to the Internet.